eclipse-theia/theia
# repo: fix `license:check` baseline

#### What it does


Currently, our [github homepage](https://github.com/eclipse-theia/theia) for the repo is reporting an unknown license.
It seems like the `license-check-baseline.json` is being interpreted as a license file and confusing github according to their [docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#detecting-a-license).

> If your repository is using a license that is listed on the Choose a License website and it's not displaying clearly at the top of the repository page, it may contain multiple licenses or other complexity. To have your license detected, simplify your LICENSE file and note the complexity somewhere else, such as your repository's README file.

<img width="311" alt="Screen Shot 2022-06-14 at 3 17 29 PM" src="https://user-images.githubusercontent.com/40359487/173673960-ee2d1547-fe98-4f90-9c80-d8b82e90466e.png">


#### How to test


1. confirm that CI is passing.
2. confirm that the `yarn license:check` command works well.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
